tests/functional/u/use/use_yield_from.txt

```diff
@@ -0,0 +1,5 @@
+use-yield-from:9:8:9:18:bad:Consider directly using 'yield from' instead:HIGH
```

Comment:
```suggestion
use-yield-from:8:4:9:18:bad:Consider directly using 'yield from' instead:HIGH
```
(Expected output in this case, if you agree with the previous comment. I'm not sure I gave the right node for the second arg in my previous suggestion.)